src/dynamics/se/dp_coupling.F90

```diff
@@ -240,14 +243,9 @@ subroutine d_p_coupling(cam_runtime_opts, phys_state, phys_tend, dyn_out)
       omega_tmp(:,:,:) = 0._r8
       phis_tmp(:,:)    = 0._r8
       q_tmp(:,:,:,:)   = 0._r8
-
-      if (cam_runtime_opts%gw_front() .or. &
-          cam_runtime_opts%gw_front_igw()) then
-
-         frontgf(:,:,:) = 0._r8
-         frontga(:,:,:) = 0._r8
-
-      end if
+      frontgf(:,:,:) = 0._r8
```

Comment:
Total nit-pick, but can we have the `= 0._r8` characters here line up with the same characters above?